docs/separating-entity-definition.md

```diff
@@ -307,7 +307,6 @@ const ASchema = new EntitySchema<A>({
     // specified on the 'discriminatorValue' property
     discriminatorValue: "my-custom-discriminator-value-for-A",
     columns: {
-        ...BaseSchema.options.columns,
```

Comment:
@pleerock @AlexMesser Could you verify these changes to the documentation and tests?
Correct me if I'm wrong, but the addition of BaseSchema columns is not needed because STI inheritance ensures the addition of BaseSchema columns using metadata from BaseSchema, so we don't need to override them.